### PR TITLE
React to Localization 3.0 changes

### PR DESCRIPTION
**Pull request by hishamco**
Address #14225

/cc @Rick-Anderson

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ryanbrandenburg  quick loc review please

File: aspnetcore/fundamentals/localization-extensibility.md [hunk 1]
```diff
@@ -41,11 +41,9 @@ The preceding providers are described in detail in the [Localization Middleware]
 
 <xref:Microsoft.AspNetCore.Localization.CustomRequestCultureProvider> provides a custom <xref:Microsoft.AspNetCore.Localization.RequestCultureProvider> that uses a simple delegate to determine the current localization culture:
 
-::: moniker range=">= aspnetcore-2.2"
-
+::: moniker range="< aspnetcore-3.0"
 ```csharp
-options.AddInitialRequestCultureProvider(
-    new CustomRequestCultureProvider(async context =>
+options.RequestCultureProviders.Insert(0, new CustomRequestCultureProvider(async context =>
```
Comment (ryanbrandenburg) [inline aspnetcore/fundamentals/localization-extensibility.md]: Why and how did these get inverted?

Comment (hishamco) [reply aspnetcore/fundamentals/localization-extensibility.md]: Because I apply prior 3.0 changes first

**Comment by hishamco** [comment]
:shipit:

**Pull request merged**